src/settings/settings.py

```diff
@@ -102,6 +102,7 @@
         "contrib.rest_framework.authentication.SimpleTokenAuthentication",
     ],
     "DEFAULT_PARSER_CLASSES": [
+        "rest_framework_xml.parsers.XMLParser",
```

Comment:
Is dit van toepassing voor alle apps? Anders misschien handiger dit specifiek aan te passen voor de reistijden_v1 app.